pkg/apps/strategy/rolling/rolling.go

```diff
@@ -223,7 +224,6 @@ func (s *RollingDeploymentStrategy) Deploy(from *corev1.ReplicationController, t
 		Timeout:         time.Duration(*params.TimeoutSeconds) * time.Second,
 		MinReadySeconds: config.Spec.MinReadySeconds,
 		CleanupPolicy:   PreserveRollingUpdateCleanupPolicy,
-		MaxUnavailable:  *params.MaxUnavailable,
```

Comment:
@tnozicka removed because we set this below if not nil... i think this worked when we did conversion from internal?

I wonder if we are not losing the default (1) or it is defaulted on server side.